src/config.rs

```diff
@@ -147,6 +147,31 @@ impl Default for RawUpdatesConfig {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
+struct RawDirectoriesConfig {
+    #[serde(default)]
+    pub custom_pages_dir: PathBuf,
+}
+
+impl Default for RawDirectoriesConfig {
+    fn default() -> Self {
+        let home_or_root = match env::var("HOME") {
+            Ok(home) => home,
+            Err(_) => String::from("/"),
+        };
+
+        Self {
+            // default to `CONFIGDIR/pages`
+            // OR `~/.local/share/tealdeer_custom_pages`
+            // OR `/.local/share/tealdeer_custom_pages`
+            custom_pages_dir: match get_config_dir() {
+                Ok(config_dir) => config_dir.join("pages"),
+                Err(_) => PathBuf::from(&home_or_root).join(".local/share/tealdeer_custom_pages"),
+            },
+        }
```

Comment:
On second thought, the config dir is probably not the correct location for this. Instead, the `UserData` directory should probably be used (see https://docs.rs/app_dirs/1.2.1/app_dirs/enum.AppDataType.html).

This can be retrieved using `get_app_root(AppDataType::UserData, &crate::APP_INFO)`. On Linux, this returns `/home/user/.local/share/tealdeer`.

(Of course, you'd still need `.join("pages")`.)